Demo/src/main/java/com/braintreepayments/demo/GooglePayFragment.java

```diff
@@ -115,7 +115,11 @@ private void handleGooglePayActivityResult(ActivityResult activityResult) {
         int resultCode = activityResult.getResultCode();
         Intent data = activityResult.getData();
         googlePayClient.onActivityResult(resultCode, data, (paymentMethodNonce, error) -> {
-            handleGooglePayActivityResult(paymentMethodNonce);
+            if (error != null) {
```

Comment:
We weren't previously handling errors in some payments flows in the demo app. I added them now to make manual testing easier (avoid app crashes)